test/integration/testsuite/prepare.go

```diff
@@ -68,15 +68,15 @@ func HandleISOVersion() {
 		GodogTags += "~cdk-only"
 		isoUrl := os.Getenv("MINISHIFT_ISO_URL")
 		switch isoUrl {
-		case "", "b2d":
-			fmt.Println("Test run using Boot2Docker iso image.")
-			isoName = "b2d"
+		case "", "centos":
+			fmt.Println("Test run using CentOS iso image.")
+			isoName = "centos"
 		case "minikube":
 			fmt.Println("Test run using Minikube iso image.")
 			isoName = "minikube"
-		case "centos":
-			fmt.Println("Test run using CentOS iso image.")
-			isoName = "centos"
+		case "b2d":
+			fmt.Println("Test run using Boot2Docker iso image.")
+			isoName = "b2d"
```

Comment:
-1, This change is not necessary because it anyway going through the switch and as per case it takes the right ISO.

Reply:
@praveenkumar for `case ""` it was setting `b2d` as the ISO. so we need to change it. We can also separate the switch case to only handle `case ""` too. WDYT?

Reply:
@praveenkumar By default when we run `make integration` the value of `isoUrl` stays empty, hence the change.

Reply:
@LalatenduMohanty Right but I don't think reorder required you can just add 
```
case "b2d" :
<>
case "", "centos":
<>
```